demo/registry.yaml

```diff
@@ -3,17 +3,20 @@ version: 1
 chains:
   musselnet:
     chain_id: musselnet-4
-    # bech32 prefix for addresses
+    # Bech32 prefix for addresses
     prefix: wasm
-    # this determines the gas payments we make (and defines the fee token)
+    # This determines the gas payments we make (and defines the fee token)
     gas_price: 0.025umayo
-    # the path we use to derive the private key from the mnemonic
+    # The path we use to derive the private key from the mnemonic
+    # Note: The hd paths shown have no meaningful relationship to the existing chains.
+    # It is recommended practice to use a different hd_path than those commonly used for user accounts.
+    # If you want this, look at the cosmoshub example below
```

Comment:
The "If you want this" confuses me as I'm not sure what "this" is
```suggestion
```
I think it'd be completely fine to remove the line. The comments below do a great job at indicating what hd_path compatibility is needed for gaia. It also gives the user more indication that the default hd_path for certain cli tools might be in a different format than `m/44'/108'/0'/1'`